Tests/GRPCTests/ConnectionPool/ConnectionPoolTests.swift

```diff
@@ -17,7 +17,7 @@
 import Logging
 import NIOCore
 import NIOEmbedded
-import NIOHTTP2
+@testable import NIOHTTP2
```

Comment:
We can't do this: it gives us access to the internals of NIOHTTP2 which is verboten.

Reply:
Removing this required copying over a simplified version of the HTTP/2 frame encoder.